ui/css/treeherder-pinboard.css

```diff
@@ -151,24 +140,12 @@
   text-decoration: underline;
 }
 
-/* Spin button suppression on chrome (bug 1045611) */
-#pinboard-related-bugs form input[type=number]::-webkit-inner-spin-button {
-  -webkit-appearance: none;
-  margin: 0;
-}
-
-/* Spin button suppression on firefox (bug 1045611) */
-#pinboard-related-bugs form input[type=number] {
-  -moz-appearance: textfield;
-}
-
```

Comment:
Just checked and we're not using a form input of type number, so this wouldn't apply in any case (though even if it did, I don't think you'd see a spinner).